src/jcampEncoder/JcampCreator.js

```diff
@@ -2,21 +2,18 @@
 /**
  * Created by acastillo on 3/2/16.
  */
+const Encoder = require('./VectorEncoder');
+const Integer = {MAX_VALUE: 2e31 - 1, MIN_VALUE: -2e31};
+const CRLF = '\r\n';
+const version = 'Cheminfo tools, March 2016';
```

Comment:
Please remove the date and instead use the version number from `require('../../package.json').version`